src/form/en_shared.jl

```diff
@@ -133,7 +133,7 @@ function constraint_mc_current_balance(pm::RectangularVoltageExplicitNeutralMode
     ungrounded_terminals = [(idx,t) for (idx,t) in enumerate(terminals) if !grounded[idx]]
 
     for (idx, t) in ungrounded_terminals
-        @smart_constraint(pm.model,  [cr, crd, crg, crs, crsw, crt, vr],
+        JuMP.@constraint(pm.model,
```

Comment:
As currently formatted, the body of the constraint is indented by a lot. (It used to match the opening `[` of the second argument. I can either leave to minimize the diff of this PR, or fix. Not sure which is best.